libs/server/Storage/Functions/MainStore/VarLenInputMethods.cs

```diff
@@ -160,10 +160,6 @@ public int GetRMWModifiedValueLength(ref SpanByte t, ref SpanByte input)
                         length += HyperLogLog.DefaultHLL.MergeGrow(srcHLL + sizeof(int), dstHLL);
                         return length + t.MetadataSize;
 
-                    case RespCommand.SETKEEPTTLXX:
```

Comment:
Is this change related somehow to the ACL implementation?

Reply:
Hmm, thought I already replied to this one...

It's not - it was a change that snuck in as I was trying to figure out how the various `SETxxx` values on `RespCommand` were used.  I'm reverting it.